pavelib/utils/test/suites/nose_suite.py

```diff
@@ -104,6 +104,7 @@ def __init__(self, *args, **kwargs):
         super(SystemTestSuite, self).__init__(*args, **kwargs)
         self.test_id = kwargs.get('test_id', self._default_test_id)
         self.fasttest = kwargs.get('fasttest', False)
+        self.use_pdb = kwargs.get('pdb', False)
```

Comment:
Now that #7714 has merged, I don't need any of this logic any more, so I've reverted it.